### ISSUE TITLE
Menubar only looks for icons in /usr/share/icons/hicolor

### ISSUE THREAD

**Issue by jaysonwillson** [awesomeWM/awesome#410]
Hello.
I realized, that menubar only shows icon for one category: internet. I have checked and found out, that this is the only category, for which icon exists in /usr/share/hicolor. After it I started to suspect something and decided to check:
echo 'naughty.notify({text = menubar.utils.lookup_icon("applications-internet.png")})' | awesome-client
returns /usr/share/icons/hicolor/48x48/apps/applications-internet.png,
but
echo 'naughty.notify({text = menubar.utils.lookup_icon("applications-multimedia.png")})' | awesome-client
returns nothing, though
locate applications-multimedia
reports, that such icon exists in
/usr/share/icons/Adwaita/(size)/categories...
/usr/share/icons/gnome/(size)/categories...
/usr/share/icons/mate/(size)/categories...
And many other places.
It seems like menubar.utils_lookup_icon only looks for category icons is /usr/share/hicolor.
Of course, I can set full paths to icons in menu_gen.lua, but it seems to me, that there should be another way, more universal.

**Comment by JulianVolodia** [comment]
I don't understand, but have time to reproduce and maybe try to fix :)

Command
```shell
echo 'naughty.notify({text = menubar.utils.lookup_icon("applications-internet.png")})' | awesome-client
```
gives me error:
> [string "naughty.notify({text = menubar.utils.lookup..."]:1: attempt to index global 'naughty' (a nil value)

Please send which verison of Awesome you use, screenshots showing actual problem.

Best regards.

**Comment by jaysonwillson** [comment]
Please forgive me, if I have not explained clearly.
I will try to be more concrete now.
In https://github.com/awesomeWM/awesome/blob/master/lib/menubar/menu_gen.lua, lines 33-54, menubar categories (which you can see when you launch menubar) are declared. 
"icon_name = "applications-multimedia.png", use = true }" string defines, which icon should be used for a specific category. menubar.utils.lookup_icon() is used to look for icons (lines 56-61)
In my case, when I launch menubar, only "Internet" category has icon, other categories do not have it. 
I have checked, if there is any difference between "applications-internet.png" icon and, for example, "applications-development.png" icon. I have realized, that "applications-internet.png" is the only icon among applications-{multimedia, development, etc...} icons used by menubar, which exists in the "hicolor" icon suite.
For example, applications-internet exists in Adwaita, Gnome, Mate, Hicolor icon suites, which are present in my system and are placed in /usr/share/icons
applications-multimedia icon exists in Adwaita, Gnome, Mate icon suites, but not in Hicolor suite, so menubar can not find it.
Here is screenshot of what I see: http://i.imgur.com/wJj9uuk.png
If echo 'naughty.notify({text = menubar.utils.lookup_icon("applications-internet.png")})' | awesome-client gives an error, please try the following:

Run awesome-client in readline mode:
awesome-client
Then run following commands inside it:
naughty = require("naughty")
menubar = require("menubar")
naughty.notify({text = menubar.utils.lookup_icon("applications-internet.png")})
naughty.notify({text = menubar.utils.lookup_icon("applications-multimedia.png")})

It will show notifications with a result of menubar.utils.lookup_icon.
For "applications-internet.png" my output is: http://i.imgur.com/OJNEc9u.png (path to this icon in Hicolor suite)
But for "applications-multimedia.png" there is no output, because this icon does not exist in Hicolor suite.

It seems to me, that menubar.utils.lookup_icon only searches for icons in /usr/share/icons/hicolor.

I use awesome from git, master branch, built on 02 August 2015. 
awesome -v:
 • Build: Aug  2 2015 19:21:27 for x86_64 by gcc version 4.9.2 (sergey@debian)
 • Compiled against Lua 5.1.5 (running with Lua 5.1)
 • D-Bus support: ✔

**Comment by jaysonwillson** [comment]
http://awesome.naquadah.org/wiki/Menubar/3.5
Setting
theme.icon_theme
in theme.lua to different values does not help.

**Comment by blueyed** [comment]
Should be basically be addressed by #468 now.
